docs/prometheus/README.md

```diff
@@ -1,114 +1,129 @@
+to indicate to Prometheus how to consume metrics data from 
+it.
 
-You can apply it directly with:
+You can apply the sample directly with the following command:
 
 ``` shell
 kubectl apply -f mongodb-prometheus-sample.yaml
 ```
 
-This will create 2 `Secrets` containing authentication for a new MongoDB user
-and Basic HTTP Auth for the Prometheus endpoint. All of this in the `mongodb`
-Namespace.
+This command creates two `Secrets` that contain authentication 
+for a new MongoDB user and basic HTTP authentication for the 
+Prometheus endpoint. The command creates both `Secrets` in the 
+`mongodb` namespace.
 
-It will also create a `ServiceMonitor` that will configure Prometheus to consume
-this resurce's metrics. This will be created in the `prometheus-system`
+This command also creates a `ServiceMonitor` that configures 
+Prometheus to consume this resource's metrics. This command 
+creates the `ServiceMonitor` in the `prometheus-system`
 namespace.
 
+## Optional: Enable TLS on the Prometheus Endpoint
 
-## Bonus: Enable TLS on the Prometheus Endpoint
+### Install Cert-Manager
 
-### Installing Cert-Manager
+1. Run the following commands to install
+   [Cert-Manager](https://cert-manager.io/) using Helm:
 
-We will install [Cert-Manager](https://cert-manager.io/) from using Helm.
+   ``` shell
+   helm repo add jetstack https://charts.jetstack.io
+   helm repo update
+   helm install \
+     cert-manager jetstack/cert-manager \
+     --namespace cert-manager \
+     --create-namespace \
+     --version v1.7.1 \
+     --set installCRDs=true
+   ```
 
-``` shell
-helm repo add jetstack https://charts.jetstack.io
-helm repo update
-helm install \
-  cert-manager jetstack/cert-manager \
-  --namespace cert-manager \
-  --create-namespace \
-  --version v1.7.1 \
-  --set installCRDs=true
-```
+2. Now with Cert-Manager installed, create a Cert-Manager 
+   `Issuer` and then a `Certificate`. You can use the two files 
+   that we provide to create a new `Issuer`:
 
-Now with Cert-Manager installed we we'll create a Cert-Manager `Issuer` and then
-a `Certificate`. We provide 2 files that can be used to create a new `Issuer`.
+   a. Run the following command to create a `Secret` holding a 
+      TLS certificate `tls.crt` and `tls.key` entries. You can 
+      use the certificate and key files that we provide in the 
+      [`testdata/tls`](https://github.com/mongodb/mongodb-kubernetes-operator/tree/master/testdata/tls) 
+      directory to create a Cert-Manager `Certificate`.
 
-First we need to create a `Secret` holding a TLS certificate `tls.crt` and
-`tls.key` entries. We provide the certificate and key files that can be used to
-create a Cert-Manager `Certificate`, they are in the `testdata/tls` directory.
+      ``` shell
+      $ kubectl create secret tls issuer-secret --cert=../../testdata/tls/ca.crt --key=../../testdata/tls/ca.key \
```

Comment:
we don't include the shell prompt `$` in the examples above. I think you can remove it here.